crates/apub/src/objects/private_message.rs

```diff
@@ -84,15 +86,26 @@ impl Object for ApubPrivateMessage {
   }
 
   #[tracing::instrument(skip_all)]
-  async fn into_json(self, context: &Data<Self::DataType>) -> LemmyResult<ChatMessage> {
+  async fn into_json(self, context: &Data<Self::DataType>) -> LemmyResult<PrivateMessage> {
     let creator_id = self.creator_id;
     let creator = Person::read(&mut context.pool(), creator_id).await?;
 
     let recipient_id = self.recipient_id;
     let recipient = Person::read(&mut context.pool(), recipient_id).await?;
 
-    let note = ChatMessage {
-      r#type: ChatMessageType::ChatMessage,
+    let instance = Instance::read(&mut context.pool(), recipient.instance_id).await?;
+    let mut kind = PrivateMessageType::Note;
+
+    // For Lemmy versions before 0.20, send private messages with old type
```

Comment:
Should probably also add a TODO, so we don't leave in these kludges forever, and can find them easier.

Reply:
Changed these to errors

Reply:
Added `deprecated` to the comment